app/code/core/Mage/Customer/Helper/Data.php

```diff
@@ -71,7 +71,7 @@ class Mage_Customer_Helper_Data extends Mage_Core_Helper_Abstract
      * WSDL of VAT validation service
      *
      */
-    const VAT_VALIDATION_WSDL_URL = 'http://ec.europa.eu/taxation_customs/vies/services/checkVatService?wsdl';
+    const VAT_VALIDATION_WSDL_URL = 'http://ec.europa.eu/taxation_customs/vies/checkVatService.wsdl';
```

Comment:
This url also gives me a redirect:
```
curl -I http://ec.europa.eu/taxation_customs/vies/checkVatService.wsdl 
 
HTTP/1.1 307 Temporary Redirect
Location: https://ec.europa.eu/taxation_customs/vies/checkVatService.wsdl
Cache-Control: no-cache
Pragma: no-cache
Content-Type: text/html; charset=utf-8
Proxy-Connection: close
Connection: close
Content-Length: 738

➜  ~ curl -I https://ec.europa.eu/taxation_customs/vies/checkVatService.wsdl

HTTP/1.1 200 OK
Date: Wed, 29 Apr 2020 06:14:56 GMT
Last-Modified: Wed, 11 Mar 2020 18:19:23 GMT
Server: Europa
Proxy-Connection: Keep-Alive
Connection: Keep-Alive

```
It should use httpS

Reply:
```suggestion
    const VAT_VALIDATION_WSDL_URL = 'https://ec.europa.eu/taxation_customs/vies/checkVatService.wsdl';
```
thanks, in my tests curl follow the https redirect but better to use https directly